website/docs/r/terraform_version.html.markdown

```diff
@@ -39,10 +39,14 @@ The following arguments are supported:
 
 ## Import
 
-Terraform versions can be imported; use `<TERRAFORM VERSION ID>` as the import ID. For example:
+Terraform versions can be imported; use `<TERRAFORM VERSION ID>` or `<TERRAFORM VERSION NUMBER>` as the import ID. For example:
 
 ```shell
 terraform import tfe_terraform_version.test tool-L4oe7rNwn7J4E5Yr 
 ```
 
+```shell
+terraform import tfe_terraform_version.test 1.1.2 
```

Comment:
Version numbers do not uniquely identify terraform versions, unfortunately, and people use them in weird ways. What do you think about an additional import string like:

`terraform import tfe_terraform_version.test official/1.1.2`

...at least then we can extend this in the future, and further narrow the potential matches by official attribute == true

Reply:
this prefix string also fixes the brittleness of splitting on "." (version numbers may not even need this!)

![Screen Shot 2022-01-14 at 9 41 48 AM](https://user-images.githubusercontent.com/174332/149552280-a15d4abd-a276-47af-bb04-a9ad353bf245.png)

Reply:
So that was my original implementation using `official/<VERSION NUMBER>` for `offical = true` and `internal/<VERSION NUMBER>` for `false`, but my understanding was that the `version` attribute is unique.

Reply:
I just checked, uniqueness is enforced by rails and I was mistaken! but splitting on "." thing is an issue.

Reply:
Yep, I was iffy on splitting by `.` (was not aware versions didn't have to follow semver!) and am open to alternative solutions. We also have to keep in mind the practitioner can supply either a tool version external ID or the version number itself so I have to have a reliable way of distinguishing between the two. I guess what we can do is split on `-` and if the first elem is `tool` then it's an external ID and passthrough.